completions/Makefile.am

```
@@ -1000,7 +1001,7 @@ symlinks: $(DATA)
 	$(ss) freeciv-server \
 		civserver
 	$(ss) _gaiacli \
-		_gaiad _npm _tendermint
+		_gaiad _npm _tendermint _ngrok
```

I've attempted to keep these sorted alpabetically, pre-approved with this change
```suggestion
		_gaiad _ngrok _npm _tendermint
```

---

Ah, right! I'll fix it. Thank you for your review.

---

2de6e1b1226c4d06fc90392dfbbb547cf523ad74, and then squashed into 3ad11012c03cee677a2080674b0056d39441676b.